procedures/foreman/release.md.erb

```diff
@@ -52,7 +52,13 @@ Note: If for some reason there was an issue with the tarballs that required uplo
 
 * [ ] Update [foreman-packaging](https://github.com/theforeman/foreman-packaging) [rpm/<%= short_version %>](https://github.com/theforeman/theforeman-rel-eng/blob/master/bump_rpm_packaging) and [deb/<%= short_version %>](https://github.com/theforeman/theforeman-rel-eng/blob/master/bump_deb_packaging) branches
 * [ ] Trigger [release_packages in Jenkins](https://ci.theforeman.org/job/release_packages/) by calling [release_packages](https://github.com/theforeman/theforeman-rel-eng/blob/master/release_packages)
+* [ ] Tag `foreman-release` into `client-rhel7`, as this doesn't happen automatically for that one tag:
+```
+koji tag-build foreman-client-<%= short_version %>-rhel7 foreman-release-VERSION.el7
```

Comment:
Don't we have the value for `version` here? It should always match the Foreman version itself and I think we can assume the release is `1` since we just incremented the version.

Reply:
not for RCs